src/core/tests/Test_CoreMain.py

```diff
@@ -1122,6 +1146,7 @@ def test_delete_temp_folder_contents_failure(self):
 
         # reset os.remove() mock
         os.remove = self.backup_os_remove
+
         runtime.stop()
```

Comment:
All changes upto this line are unrelated to this PR. See my other comment on what can be done with these

Reply:
the line break is added because the file format is not consistent, but this is very small change. when i merge in master's changes i will revert all the formatting changes if you feel strongly about them

Reply:
Ignore this for now to look at the bigger picture. Rajasi's guidance generally applies but I'm resolving to reduce noise.

Reply:
Reopening -- Please address this comment as it's making it very difficult to review this code scrolling up and down.

Reply:
done